vizro-core/docs/pages/user_guides/tabs.md

```diff
@@ -0,0 +1,128 @@
+# How to use tabs
+
+Tabs organize and separate groups of related content in a dashboard, letting users navigate between different sections or views within a click.
+This guide shows you how to use tabs to organize your containers into subsections inside the dashboard.
+
```

Comment:
We should have something here that explains what I was trying to say in https://github.com/McK-Internal/vizro-internal/issues/292#issuecomment-1866153211 about how tabs are really just a way of stacking containers. Getting this mental model clear is very important because it will help people understand how simple it is to configure a dashboard and how much sense (hopefully) it makes, rather than them feeling like they're needing to learn more and more concepts every time they want to change layout slightly.

Reply:
Likewise we should have something that links from the containers page to here with something like "an alternative way of displaying multiple containers is to put them inside `Tabs` blah blah". Don't worry if you feel like you're repeating the same point in multiple places. People learn by repetition, and it's useful for a user to end up on a single documentation page and be linked to other ones that will help them understand related topics easily. We need to make sure that our docs are a cohesive whole rather than just a set of unrelated pages.

Reply:
Let me know if the current changes convey this sufficiently, but I definitely get your point 👍 

https://github.com/mckinsey/vizro/pull/282/commits/a1483f7b355faab846c8766f6cc64d987a85c7b4